test.sh

```diff
@@ -75,6 +75,16 @@ for dir in ${VERSIONS}; do
       echo "-> Upstream tests are not present, skipping"
     fi
   fi
+
+  if [ -n "${TEST_OPENSHIFT_PYTEST}" ]; then
+    if [[ -x test/run-openshift-pytest ]]; then
```

Comment:
We shouldnt need double brackets over here

Reply:
The double brackets are used in the whole script it makes sense to remove it in the entire script. We do not use any enhanced functionality.

Reply:
https://github.com/sclorg/container-common-scripts/pull/373/commits/9fdd56c9924ec202f8e8a57c8fabcb984cde5599